### ISSUE TITLE
[css-paint-api] Expose DOMMatrix etc

### ISSUE THREAD

**Issue by jakearchibald** [w3c/css-houdini-drafts#1050]
https://drafts.fxtf.org/geometry/ - these functions and classes are useful when working with points in 2d drawing. It feels like they should be available in a paint worklet.

**Comment by tabatkins** [comment]
Agree, but shouldn't this then be a bug against the Geometry spec to add the right `[Exposed]` annotations?

**Comment by jakearchibald** [comment]
Yeah, probably. I couldn't decide, since the paint worklet spec exposes devicePixelRatio itself.
